valhalla/baldr/directededge.h

```diff
@@ -1044,8 +1086,11 @@ class DirectedEdge {
   json::MapPtr json() const;
```

Comment:
Might want to add this to the JSON output.  Perhaps conditional (e.g. if speed > 0 add it)?

Reply:
no conditional, but added.